src/test/java/org/sqlite/ConnectionTest.java

```diff
@@ -50,13 +50,16 @@ public void executeUpdateOnClosedDB() throws SQLException {
     }
 
     @Test
-    public void readOnly() throws SQLException {
-
+    public void readOnly() throws SQLException, IOException {
         // set read only mode
         SQLiteConfig config = new SQLiteConfig();
         config.setReadOnly(true);
 
         try (Connection conn = DriverManager.getConnection("jdbc:sqlite:", config.toProperties())) {
+            assertThat(((SQLiteConnection) conn).getDatabase().isReadOnly("main")).isTrue();
+            assertThat(((SQLiteConnection) conn).getDatabase().isReadOnly(null)).isTrue();
+            assertThat(((SQLiteConnection) conn).getDatabase().isReadOnly("temp")).isFalse();
+            assertThat(((SQLiteConnection) conn).getDatabase().isReadOnly("false")).isFalse();
```

Comment:
what is this `"false"` database ?

Reply:
> but "false" made sense too since the result should be "false".

i feel this is confusing, a more explicit string would be best

Reply:
ok, i'll update this name when i make any other changes that we decide on in sibling comments.

Reply:
handled by new commit